tidy3d/plugins/dispersion/fit.py

```diff
@@ -448,9 +489,144 @@ def plot(
 
         return ax
 
+    @staticmethod
+    def validate_url_load(data_load: List):
+        """Validate if the loaded data from URL is valid
+            The data list should be in this format:
+                [["wl",     "n"],
```

Comment:
can we see whether this looks ok in the docs?

Reply:
in the from_url, now I format it as `example`